src/Views/Results/Components/SuggestedSearch/Default.cshtml

```diff
@@ -0,0 +1,27 @@
+@model List<SuggestedSearchViewModel>
+
+<h3 class="heading-medium">Suggested searches</h3>
+
+@if(Model.Any()) {
+    <p>For business studies there are:</p>
+    <ul class="list list-bullet">
+
+    @foreach (var item in Model)
+    {
+      var courseText = " course" + (item.TotalCount == 1 ? "" : "s");
+      var areaText = item.ResultsFilter.RadiusOption.HasValue ?
+        " with " + item.ResultsFilter.GetFundingTypes().First().ToLowerInvariant() + " available within " + item.ResultsFilter.rad + " miles of " + item.ResultsFilter.loc :
```

Comment:
When the financial support option being linked to is 'student finance and salaries' (ie All) can we drop the "with student finance and salaries available" bit of the text:
```diff
-9 courses with student finance and salaries available within 50 miles of Brighton, UK
+9 courses within 50 miles of Brighton, UK
```